ConfigurationSystem/Client/Helpers/Operations.py

```diff
@@ -24,20 +24,63 @@
 
       Operations().getValue('someSection/aSecondOption')
         - aSecondValueInProduction if we are in 'Production' setup
-        - aSecondValue if we are in 'Certification' setup     <- looking in Default since there's no Certification/someSection/aSecondOption
+        - aSecondValue if we are in 'Certification' setup    <- looking in Default
+                                                                since there's no Certification/someSection/aSecondOption
+
+
+    At the same time, for multi-VO installations, it is also possible to specify different options per-VO,
+    like the following::
+
+      Operations/
+          aVOName/
+              Default/
```

Comment:
It is also wrong in the existing doc above; the section is called `Defaults`

Reply:
Thanks, corrected.